www/themes/unl/html/SiteMaster/Core/Controller.tpl.php

```diff
@@ -1,22 +1,34 @@
 <?php
-\UNL_Templates::$options['version']        = 4.0;
-\UNL_Templates::$options['sharedcodepath'] = dirname(__FILE__).'/sharedcode';
```

Comment:
This reminds me about something I should document. The new templates package works out-of-the-box without a copy of the WDN includes. The previous release automatically failed back to GitHub, which was not ideal. The new package has the `setLocalIncludePath` method to specify the path to the directory that contains the synced `wdn` directory. This replaces the static configuration `Templates::$options['templatedependentspath']`.

You _can_ use ` $page->setLocalIncludePath('https://unlcms.unl.edu');` to use UNLcms's synced directory. But that will require a lot of network requests for _every_ request to your site. It's best to set this to a locally managed directory.